src/arrayMethodSort.js

```diff
@@ -4,9 +4,28 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function (
+    compareFunction = (a, b) => {
+      if (a.toString() > b.toString()) {
+        return 1;
+      } else {
+        return -1;
+      }
+    },
+  ) {
+    let saveValue;
+
+    for (let j = 1; j < this.length; j++) {
+      for (let i = 1; i < this.length; i++) {
```

Comment:
The inner loop should start at index 0 to correctly compare the first pair of elements in the array.